src/handbook/peopleops/expenses.md

```diff
@@ -43,7 +43,29 @@ company, you can claim back the money through [Deel](https://help.letsdeel.com/h
 * Adjustments approved before the payroll cut-off date will be processed in the current month's payroll. Any adjustment approved after the payroll cut-off date will be processed in the next month's payroll
 * The People Ops Manager is authorized to approve expenses that are $250 or less. However, if an expense exceeds $250, both the CEO and the People Ops Manager will need to review and approve it.
 
-## Asset purchases over $500
+## Equipment
```

Comment:
Do we need a note that the company might require return of it's property if the employee leaves the company?

Reply:
Added via https://github.com/FlowFuse/website/pull/1301/commits/80505c5c237c3fbad6ed18deacadecdc1715452e